l10n_it_fatturapa_out/models/company.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Roberto Fichera <roberto.fichera@levelprime.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    max_invoice_in_xml = fields.Integer(
+        string='Default max num. invoice in XML',
```

Comment:
Anche qui, meglio compattare. 
Forse potresti omettere "Default", dovrebbe essere chiaro dal contesto (impostazioni generali) ed è specificato nell'help.
```suggestion
        string='Max Invoice # in XML',
```

Reply:
ok